app.js

```diff
@@ -58,10 +58,11 @@ function runNpm(command) {
 
 try {
 	require('sugar');
+	require('es6-shim');
```

Comment:
We only use `Map` and `Set`, right? Let's just test for the existence of `global.Set` before deciding whether or not to shim it?